### DISCUSSION TITLE
What measures of tone quality can we capture?

### DISCUSSION THREAD

**Discussion by bhp1** [edwardkort/WWIDesigner#5] [General]
Beyond note pitch (frequency), what indicators of the quality of sound can we predict with the modelling capabilities available to us?

**Comment by bhp1** [comment]
Within a linear impedance-based model, I think the following are within reach:
- quality of resonance, using Q factors
- approximate loudness (very approximate)
- cutoff frequency

I suspect that harmonic content, other than just the value of cutoff frequency, would require non-linear models of the reed or jet drive. These may be beyond us at the moment, and may be beyond the current state of the art.

**Comment by antoinelefebvre** [comment]
We may consider the harmonicity of the resonances, that is, how close is the n'th resonance to n time the first.

If we want to work on harmonic content, we can try the harmonic balance method:
http://www.lma.cnrs-mrs.fr/logiciels/harmbal/index_english.html

Or time domain synthesis.

I don't think we want to go this route at this point.

**Comment by edwardkort** [comment]
Great stuff. In the best of all possible worlds, I would include all of the above. I have been particularly intrigued by the playability calculations implemented by Paul Dickens (simple harmonic analysis) and the psychoacoustics discussions by Sandra Leon (harmonic balance).

However, the accuracy of these calculations will depend on how well we can model the sound mechanisms (jets, reeds, etc.) on the various instruments we incorporate.
